aws-sdk-core/lib/aws-sdk-core/cloudfront/signer.rb

```diff
@@ -0,0 +1,149 @@
+require 'base64'
+require 'uri'
+require 'time'
+require 'json'
+require 'openssl'
+
+module Aws
+  module CloudFront
+
+    # Allows you to create signed URLs for Amazon CloudFront resources
+    #
+    #     signer = Aws::CloudFront::UrlSigner.new
+    #     url = signer.signed_url(url,
```

Comment:
Since `#signed_url` has been removed from this class, could you make changes at documentations as well?